.circleci/config.yml

```diff
@@ -27,6 +27,10 @@ jobs:
           name: Bundle Install
           command: bundle install
 
+      - run:
+          name: Install libsodium
+          command: sudo apt-get install libsodium-dev
```

Comment:
Necessary to pick up porky_lib changes on daffy_lib.

Reply:
is this needed on all projects using porky_lib?

Reply:
In the process of finding that out.  Saw Foghorn and Wile merge fine.  But Roadrunner is complaining as well, and this does not seem to be enough.  Will need to dig into that one further.